contracts/libraries/TransferHelper.sol

```diff
@@ -51,4 +51,13 @@ library TransferHelper {
             address(token).call(abi.encodeWithSelector(IERC20Minimal.transferFrom.selector, from, to, value));
         require(success && (data.length == 0 || abi.decode(data, (bool))), "STF");
     }
+
+    /// @notice Transfers ETH to the recipient address
+    /// @dev Fails with `STE`
+    /// @param to The destination of the transfer
+    /// @param value The value to be transferred
+    function safeTransferETH(address to, uint256 value) internal {
```

Comment:
added this because it's used in `PeripheryPayments`

Reply:
consider using solmate's impl https://github.com/transmissions11/solmate/blob/main/src/utils/SafeTransferLib.sol